types/hashids/index.d.ts

```diff
@@ -29,6 +29,3 @@ declare namespace Hashids {
         unhash(input: string[], alphabet: string): number;
     }
 }
-
-declare var hashids: Hashids.IHashids;
```

Comment:
although you don't need `declare module 'hashids'` anymore, as TS 2.0 is the defacto now. just export the default class, and if you want, `declare global { const Hashids: Hashids; }`